docker/runtime/ruby-2.4/event-trigger/kubeless.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
```

Comment:
I am stickler for ASL v2.0 headers

Reply:
btw the shebang `#!` line should be the 1st in the script, see e.g. https://en.wikipedia.org/wiki/Shebang_(Unix)

Reply:
good spot, I'll update